pallets/subtensor/src/lib.rs

```diff
@@ -1903,6 +1903,15 @@ pub mod pallet {
             Self::do_root_register(origin, hotkey)
         }
 
+        /// Attempt to adjust the senate membership to include a hotkey
+        #[pallet::call_index(63)]
+        #[pallet::weight((Weight::from_parts(0, 0)
+		.saturating_add(T::DbWeight::get().reads(0))
+		.saturating_add(T::DbWeight::get().writes(0)), DispatchClass::Normal, Pays::No))]
```

Comment:
Pay::No is fine for call from sudo. But the extrinsic can be called by any coldkey. The use may attach the chain by sending too many extrinsics.

Reply:
I think this could be a pays::yes
```suggestion
		.saturating_add(T::DbWeight::get().writes(0)), DispatchClass::Normal, Pays::Yes))]
```

Reply:
Not sure if the weights will be auto calculated later. At lease, the basic extrinsic fee will be charged if Pays::Yes.